.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
         with:
           tool: cargo-hack,taplo-cli
 
+      - name: Check Markdown
```

Comment:
nit: you can just remove this; we don't have a name for the taplo action right below

Reply:
We should have a name for the taplo action, because it's not immediately obvious what taplo is or what it's doing.

Reply:
Want to add it?

Reply:
Done